src/components/study-container.js

```diff
@@ -309,13 +309,12 @@ export function StudyContainer({ view, onChangeTab }) {
                         }
                     })
                     .catch((err) => {
-                        console.error(err.message);
+                        console.error(err);
                         displayErrorMessageWithSnackbar({
-                            errorMessage: err.message,
+                            errorMessage: err,
```

Comment:
You could use `useSnackMessage` custom hook here

Reply:
Done